deploy/default-config.yaml

```diff
@@ -0,0 +1,10 @@
+kind: DevWorkspaceOperatorConfig
+apiVersion: controller.devfile.io/v1alpha1
+metadata:
+  name: devworkspace-operator-config
+config:
+  routing:
+    clusterHostSuffix: ${ROUTING_SUFFIX}
```

Comment:
This actually shows another issue we have: it would be better to report: `Waiting for main endpoint be ready`, health check fails:
![Screenshot_20210920_145448](https://user-images.githubusercontent.com/5887312/133998078-37505271-d131-40ce-b339-fd13a3e6b176.png)

it makes sense to create a separate issue for that. I'll do

Reply:
Probably the background for one more issue:
After clusterSuffix is cleaned up, it's not propagated to existing workspaces, even after restart. Removing DevWorkspace Routing helps but it's not desired behaviour I think.

Reply:
Hmm I'm not sure how to reproduce the above issue (deployment is running but devworkspace reports `waiting for workspace deployment`). When I start a DevWorkspace locally, I see the `Waiting for editor to start` message (sometimes, only briefly) -- I assume if startup is really hung on the health check, that message should show up.

Reply:
> After clusterSuffix is cleaned up, it's not propagated to existing workspaces, even after restart. Removing DevWorkspace Routing helps but it's not desired behavior I think.

This is because the DevWorkspaceRouting CR doesn't store `clusterRouteSuffix`, so reconciling the DevWorkspace won't trigger any reconciles to the DevWorkspaceRouting. Anything triggering a reconcile for the routing object will cause the change to be reflected (e.g. modifying annotations/labels on routes) but otherwise there are no events for the routing controller to respond to.

We could work around this by propagating the `.spec.started` field to `DevWorkspaceRouting`; this would at least enable stopping + restarting the workspace to propagate the config change.

Reply:
Created https://github.com/devfile/devworkspace-operator/issues/602